<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FItemTagsEdit</class>
 <widget class="QDialog" name="FItemTagsEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tags edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/hacc.qrc">
    <normaloff>:/items/tag.png</normaloff>:/items/tag.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="containerLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tbTagAdd">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/dialog-ok.png</normaloff>:/main/icons/dialog-ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRevert">
       <property name="text">
        <string>Revert</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-undo.png</normaloff>:/main/icons/edit-undo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/dialog-ok-apply.png</normaloff>:/main/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/dialog-cancel.png</normaloff>:/main/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>FItemTagsEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
